README.rst

```diff
@@ -35,17 +38,21 @@ TOC partitions should have the following fields:
 Field Name           Type    Required  Description
 -------------------  ------  --------  ---------------------------------------------------------------------------------
 version              string  Yes       Semantic versioning of the file format.
-tocs                 list    Yes       List of relative paths or URLs to other TOC files.
+tocs                 list    Yes       Map of names to relative paths or URLs of `image partitions`__ or
```

Comment:
I think this should be type `dict` not `list`

Reply:
fixed.